packages/frontend/src/js/app/components/map/map.js

```diff
@@ -881,82 +890,75 @@ export default {
-                    this.numberOfShantytownsBy.regions[
-                        feature.properties.code
-                    ] || 0;
-                if (nbSites > 0) {
-                    this.circleWithText(
-                        this.map,
-                        [lat, lon],
-                        this.numberOfShantytownsBy.regions[
-                            feature.properties.code
-                        ] || 0,
-                        20,
-                        3,
-                        "region"
-                    ).addTo(this.regionalLayer);
-                    this.regionalLayer.setStyle({
-                        color: "white",
-                        weight: 8,
-                        opacity: 1
-                    });
-                }
+                    this.numberOfShantytownsBy.regions[key].sites > 0
+                        ? this.numberOfShantytownsBy.regions[key].sites
+                        : 0;
+                this.circleWithText(
+                    this.map,
+                    [
+                        this.numberOfShantytownsBy.regions[key].latitude,
+                        this.numberOfShantytownsBy.regions[key].longitude
+                    ],
+                    [
+                        this.numberOfShantytownsBy.regions[key].chieftown
+                            .latitude,
+                        this.numberOfShantytownsBy.regions[key].chieftown
+                            .longitude
+                    ],
+                    `<div>${nbSites}</div>`,
+                    20,
+                    3,
+                    "region"
+                ).addTo(this.regionalLayer);
             });
         },
 
-        // Fonction de chargement des données geoJson départementales
+        // Fonction de chargement des marqueurs départementaux
         loadDepartementalData() {
             this.departementalLayer.getLayers().forEach(layer => {
                 if (layer instanceof L.Marker) {
                     layer.remove();
                     return;
                 }
-
-                // Calcul de la position du marqueur
-                // On utilise la méthode pointOnFeature(), qui garantit que le point soit dans le polygone, plutôt que centroid()
-                const { feature } = layer;
-                const markerPosition = pointOnFeature(feature);
-                // Création du marqueur à partir de la long et lat retournées par pointOnFeature()
-                const lon = markerPosition.geometry.coordinates[0];
-                const lat = markerPosition.geometry.coordinates[1];
-
-                const nbSites =
-                    this.numberOfShantytownsBy.departements[
-                        feature.properties.code
-                    ] || 0;
-                const siteLabel = nbSites > 1 ? "sites" : "site";
-                if (nbSites > 0) {
+            });
+            Object.keys(this.numberOfShantytownsBy.departements).forEach(
```

Comment:
Même remarques que pour les régions :
- ça pourrait être plus lisible en gettant les propriétés de `this.numberOfShantytownsBy.departements[key]` en haut de la boucle
- la condition sur `this.numberOfShantytownsBy.departements[key].sites` me paraît étrange